docs/tutorials/gs-continuous-space.ipynb

```diff
@@ -0,0 +1,988 @@
+{
```

Comment:
<p>can we try to remove the warning ? </p><p>/home/gabriel/Documents/temp/venv/lib/python3.10/site-packages/netket/vqs/mc/mc_state/state.py:252: UserWarning: n_samples=1000 (1000 per MPI rank) does not divide n_chains=16, increased to 1008 (1008 per MPI rank) </p><p>  self.n_samples = n_samples </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>